src/components/HomeHeader.js

```diff
@@ -1,184 +1,200 @@
                   <img
-                    alt='organization logo'
-                    src={process.env.PUBLIC_URL + '/icons/litmus.png'}
-                    className='logo-img'
+                    alt="organization logo"
+                    src={process.env.PUBLIC_URL + "/icons/litmus.png"}
+                    className="logo-img"
                   />
                 </div>
               </a>
-              <Link to={process.env.PUBLIC_URL} className='home'>
-                <h3 className='top-header-home'>{<FaHome />}</h3>
+              <Link to={process.env.PUBLIC_URL} className="home">
+                <h3 className="top-header-home">{<FaHome />}</h3>
               </Link>
               <input
-                className='top-header-input'
-                placeholder='Search for charts...'
+                className="top-header-input"
+                placeholder="Search for charts..."
                 value={this.state.searchTerm}
                 onChange={this.handleSearchTermChange}
               />
               <a
-                href='https://github.com/litmuschaos/community-charts/blob/master/CONTRIBUTING.md'
-                target='_blank'
-                rel='noopener noreferrer'
+                href="https://github.com/litmuschaos/community-charts/blob/master/CONTRIBUTING.md"
+                target="_blank"
+                rel="noopener noreferrer"
               >
-                <h3 className='top-header-contribute'>
+                <h3 className="top-header-contribute">
                   Contribute
-                  <span className='contribute-icon-container'>
+                  <span className="contribute-icon-container">
                     {/*<FaChevronDown />*/}
                   </span>
                 </h3>
               </a>
               <a
-                href='https://docs.litmuschaos.io/'
-                target='_blank'
-                rel='noopener noreferrer'
+                href="https://docs.litmuschaos.io/"
+                target="_blank"
+                rel="noopener noreferrer"
               >
-                <h3 className='top-header-contribute'>
-                  Docs<span className='contribute-icon-container'></span>
+                <h3 className="top-header-contribute">
+                  Docs<span className="contribute-icon-container"></span>
                 </h3>
               </a>
-              <Dropdown options={this.props.versions} onChange={this.changeVersion} value={getVersion(this.props.versions)}/>
+              <Dropdown
+                options={this.props.versions}
+                onChange={this.changeVersion}
+                value={getVersion(this.props.versions)}
+              />
+
+              <div
+                className="top-header-theme-changer"
+                onClick={() => this.props.toggleTheme()}
+              >
+                {this.props.isDarkTheme ? <FaMoon size={25} color='#FFFFFF' /> : <FaSun size={25} color='#FFD600' />}
```

Comment:
Please Add, Tooltip text, on mouse hovering "Dark Mode" and "Light Mode".

Reply:
Ok will add tooltips

Reply:
Done.